backend/.env.dev

```diff
@@ -1,9 +1,11 @@
-DEBUG=1
+DEBUG=0
 SECRET_KEY=foo
-DJANGO_ALLOWED_HOSTS=localhost 127.0.0.1 [::1] 0.0.0.0 159.89.9.110
+DJANGO_ALLOWED_HOSTS="localhost 127.0.0.1 [::1] 0.0.0.0 159.89.9.110 104.248.32.131"
```

Comment:
Docelowo i tak chcemy wrzucić to na serwer na uczelni, czy chcemy te adresy do digitalocean? @xszym ?